medicinal_plants/lib/src/screens/landing-screen.dart

```diff
@@ -39,18 +41,66 @@ class LandingScreen extends StatelessWidget {
             ),
           ),
 
-          Container(
-            margin: EdgeInsets.zero,
-            padding: EdgeInsets.zero,
-            width: width,
-            height: height,
+          Center(
             child: Text(
               'Medicinal Plants',
-              style: TextStyle(
-                
+              style: GoogleFonts.poppins(
```

Comment:
I think most of the fonts we will be using is Poppins,
create a folder inside constants folder, then create a file:  medicinal_themes.dart
and make the TextStyle reusable:

```
whiteTextStyle({required double size, required FontWeight weight})
{
    return GoogleFonts.poppins(
        textStyle: TextStyle(
            color: Colors.White,
            fontSize: size,
            fontWeight: weight,
        )
    );

}
```